tools/ImporterV2/App.config

```diff
@@ -0,0 +1,52 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<configuration>
+  <!--
+  This code allows to import resources from a storage container to a FHIR service.
+  Input data must be provided in ndjson format in one container in Azure blobs storage.
+  -->
+  <appSettings>
+    <!-- 
+    If you just want to monitor an import then provide a url for this key/value
+    Note that if this is provided then it only performs this operation and will not do any import.
+    https://{your fhir endpoint}/_operations/import/{import id}
+    The expected endpoint would be similar to this:
+    -->
+    <add key="MonitorImportStatusEndpoint" value=""/>
+    
+    <!--
+    The amount of time to delay between calls to get the import status
+    The value should be a timespan value
+    -->
+    <add key="ImportStatusDelay" value="00:02:00"/>
+
+    <!-- Azure blob storage parameters -->
+    <add key="ConnectionString" value="{Add your BlobServiceClient connection string here}" />
+    <add key="ContainerName" value="{Add your container}" />
+    <!-- 
+    the type of files it'll search for in the container 
+    do not add a file extension as ndjson is assumed and added to the ResourceType automatically
+    -->
+    <add key="ResourceType" value="Observation" />
```

Comment:
><add key="ResourceType" value="Observation" /> [](http://example.com/codeflow?start=4&length=46)

Please specify how to handle case when I don't want to have any filters.

Reply:
I updated the comment stating that leaving this empty will search all.